templates/server/src/main/resources/archetype-resources/src/main/java/__packageInPathFormat__/general/common/to/AbstractCto.java

```diff
@@ -0,0 +1,19 @@
+package ${package}.general.common.to;
+
+/**
+ * This is the abstract base class for a <em>CTO</em> (composite {@link AbstractTo transfer-object}). Such object should
+ * contain (aggregate) other {@link AbstractTo transfer-object}s or {@link java.util.Collection}s of those. However, a
+ * CTO shall never have properties for atomic data (datatypes).<br>
+ * Classes extending this class should carry the suffix <code>Cto</code>.
+ */
+public abstract class AbstractCto extends AbstractTo {
```

Comment:
Again I would not create all of this Eto, Cto ... types. In my eyes that's something the implementer needs to decide. We should better document it as a good pattern to use and explain it

Reply:
According to our different PoV should we create two archetypes?
* minimal app with almost no code
* app for projects to get started solving all the features we considered as basic foundation in devon4j so far